test/runtests.jl

```diff
@@ -2,179 +2,103 @@
-    @test_throws ArgumentError vec_tbl_csv(99942, Date(2000), Date(2010), Year(1); CENTER="nomatch")
-    @test_throws ArgumentError vec_tbl_csv(499, Date(2009), Date(2010), Year(1); VEC_TABLE = 1, CENTER="mars")
-    dt0 = Date(2000); dtmax = Date(2015); δt = Year(1)
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, δt; VEC_TABLE = "2xa", CENTER="coord", COORD_TYPE="w")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, δt; VEC_TABLE = "2xa", CENTER="coord", COORD_TYPE="%")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, δt; VEC_TABLE = "2xa", CENTER="coord", COORD_TYPE="")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, δt; VEC_TABLE = "2xa", CENTER="coord", COORD_TYPE="c", SITE_COORD="a,b,c")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, δt; VEC_TABLE = "2xa", CENTER="coord", COORD_TYPE="c", SITE_COORD="")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, δt; REF_PLANE="T", VEC_TABLE = "2xa", CENTER="coord", COORD_TYPE="c", SITE_COORD="10,1,1")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", "1400-1-1", dtmax, δt)
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", "bad-start-time", dtmax, δt)
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, "3500-2-1", δt)
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, "bad-stop-time", δt)
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, "1 w")
-    @test_throws ArgumentError vec_tbl_csv("1950 DA", dt0, dtmax, "bad-step_size")
-    @test_throws TypeError vec_tbl_csv("1950 DA", dt0, dtmax, δt; VEC_LABELS=0)
-    @test_throws TypeError vec_tbl("1950 DA", dt0, dtmax, δt; VEC_LABELS=0)
+@testset "Small-Body DataBase API" begin
+    # Search 433 Eros in three different ways 
+    eros_1 = sbdb("sstr" => "Eros")
+    eros_2 = sbdb("spk" => "2000433")
+    eros_3 = sbdb("des" => "433")
+    
+    @test eros_1 == eros_2 == eros_3
+    @test isa(eros_1, Dict{String, Any})
+    @test isa(eros_1["orbit"], Dict{String, Any})
+    @test isa(eros_1["signature"], Dict{String, Any})
+    @test isa(eros_1["object"], Dict{String, Any})
+
+    # Search Apophis's close approach data
+    apophis = sbdb("sstr" => "Apophis", "ca-data" => "true")
+    @test isa(apophis, Dict{String, Any})
+    @test isa(apophis["orbit"], Dict{String, Any})
+    @test isa(apophis["signature"], Dict{String, Any})
+    @test isa(apophis["object"], Dict{String, Any})
+    @test isa(apophis["ca_data"], Vector{Any})
 end
+
+@testset "Small-Body Radar Astrometry API" begin
+    # Search Apophis' radar astrometry
+    apophis_1 = sbradar("spk" => "20099942")
+
+    @test isa(apophis_1, Dict{String, Any})
+    @test isa(apophis_1["fields"], Vector{Any})
+    @test isa(apophis_1["signature"], Dict{String, Any})
+    @test isa(apophis_1["data"], Vector{Any})
+    @test isa(apophis_1["count"], String)
+    @test length(apophis_1["data"]) == parse(Int, apophis_1["count"])
+  
```

Comment:
I think around here we can add some tests to check the output, e.g.:
```suggestion
    apophis_1["fields"] == ["des", "epoch", "value", "sigma", "units", "freq", "rcvr", "xmit", "bp", "observer"]
    all(first.(apophis_1["data"]) .== "99942")
    apophis_1["signature"]["source"] == "NASA/JPL Small-Body Radar Astrometry API"
    VersionNumber(apophis_1["signature"]["version"]) ≥ v"1"
```

Reply:
(EDIT: added `@test` to suggestions above)